src/components/DocumentationTopic.vue

```diff
@@ -367,6 +361,18 @@ export default {
   @include dynamic-content-container;
 }
 
+.description {
+  &:empty { display: none; }
+
+  &:not(:empty) {
+    margin-bottom: $contenttable-spacing-single-side;
+  }
+
+  /deep/ .content + * {
```

Comment:
I was moving the CSS from `Description` component and wasn't sure if we still need this? It seems like the `DeprecationSummary` component overwrote margin-top; `RequirementMetadata` doesn't have `.content`; I couldn't find an example for `downloadNotAvailableSummary` @mportiz08

Reply:
I think another scenario where this rule would trigger is on a sample code page where the button matches this selector as the first thing following the abstract content. However, it looks like the button's own margin has a higher priority, so this isn't actually applied in that case.

I did double check a page with a deprecation aside, and it looks like this rule _is_ being used for that top margin however.